srtcore/congctl.h

```diff
@@ -189,7 +189,7 @@ class SrtCongestionControlBase
 
     // Called when the settings concerning m_llMaxBW were changed.
     // Arg 1: value of CUDT::m_llMaxBW
-    // Arg 2: value calculated out of CUDT::m_llInputBW and CUDT::m_iOverheadBW.
+    // Arg 2: value calculated out of CUDT::m_config::llInputBW and CUDT::m_config::iOverheadBW.
```

Comment:
What is `CUDT::m_config::...`? I believe you wanted to fix into `CSrtConfig::...`.